### DISCUSSION TITLE
What's stopping V UI from reaching 0.0.5?

### DISCUSSION THREAD

**Discussion by Wajinn** [vlang/v#14423] [General]
It seems like it's being continually updated, but not enough to make it to 0.0.5.  Out of curiosity, what is the criteria for it to reach that point (0.0.5)?

**Comment by larpon** [comment]
I don't think UI follows any strict versioning scheme or roadmap currently.

**Reply by Wajinn** [reply]
Well, it gets a bit confusing.  You can see that improvements are being made (and that's great), but it's not quite clear where the project is going or what goals it's trying to achieve next.

**Comment by medvednikov** [comment]
we'll release V UI 0.1 after V 0.3 (which will be released this week)
